app/controllers/games_controller.rb

```diff
@@ -1,13 +1,17 @@
 # Main controller for chess session for CRUD
 # logic for our app
 class GamesController < ApplicationController
-  before_action :authenticate_user!, only: [:create, :new]
+  before_action :authenticate_user!, only: [:create]
 
   def index
+    @games = Game.all
   end
 
   def create
-    @game = Game.new
+    @game = Game.create
+    @game.white.update(user_id: current_user.id)
```

Comment:
This is performing three db writes and is inefficient. The game should be created and populated with the player all in one fell swoop.